opensquirrel/utils/check_mapper.py

```diff
@@ -39,7 +40,7 @@ def check_mapper(mapper: Mapper) -> None:
 
     ir = IR()
     ir.add_gate(BlochSphereRotation(42, (1, 0, 0), 1, 2))
-    ir.add_gate(ControlledGate(42, BlochSphereRotation.identity(100)))
+    ir.add_gate(ControlledGate(42, I(100)))
```

Comment:
This function looks like a test more than a utility function, can you explain why these numbers are important?

Reply:
I cannot because I didn't write the function. I have only changed `BlochSphereRotation.identity` for `I` because I have removed `BlochSphereRotation.identity`.

Reply:
Can you see if it necessary to have in the function, if not, do you think it's best to move to a test?